tests/test_config.py

```diff
@@ -0,0 +1,33 @@
+# Store some global configuration for tests
+
+import numpy as np
+
+_core_supported_dtypes = [
+    np.float16,
+    np.float32,
+    np.float64,
+    np.int16,
+    np.int32,
+    np.int64,
+    np.uint16,
+    np.uint32,
+    np.uint64,
+]
+
+supported_dtypes_x = _core_supported_dtypes + [np.datetime64]
```

Comment:
Can we also support `np.timedelta64` as both `x` and `y`

Reply:
We can also make an issue for this and leave this for a future PR.

Reply:
Fair remark! Can indeed think of some use cases!